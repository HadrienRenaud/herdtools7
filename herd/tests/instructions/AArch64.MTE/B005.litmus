AArch64 B005
Variant=mte,sync,faultToNext
{
 0:X1=x:red;
}
 P0          ;
 LDR W0,[X1] ;
 MOV W2,#1   ;
forall(0:X2=1)
