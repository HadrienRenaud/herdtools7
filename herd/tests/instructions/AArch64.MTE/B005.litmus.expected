Test B005 Required
States 1
0:X2=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X2=1)
Observation B005 Always 1 0
Hash=92ce7c8ed051bc37bffd09f2abe7bb5d

